### DISCUSSION TITLE
Just Thanks!

### DISCUSSION THREAD

**Discussion by JoshWay** [aleksamagicka/aquacomputer_d5next-hwmon#99] [General]
Bro!  I'm so stoked to have found that you did this work. I hope others can jump in and help build on it.  If I were a programmer, I would have done this many years ago!

**Comment by aleksamagicka** [comment]
Thanks! This project wouldn't be what it is [without others helping](https://github.com/aleksamagicka/aquacomputer_d5next-hwmon/graphs/contributors) 😄
